src/locale/locales/id/messages.po

```diff
@@ -3646,15 +3646,15 @@ msgstr "Tampilkan Kutipan Postingan"
 
 #: src/screens/Onboarding/StepFollowingFeed.tsx:118
 msgid "Show quote-posts in Following feed"
-msgstr ""
+msgstr "Tampilkan kutipan postingan di feed Mengikut"
```

Comment:
```suggestion
msgstr "Tampilkan kutipan postingan di feed Mengikuti"
```
Sorry, I didn't check thorough on my previous review. But, there is a misspelling word `mengikuti` in line 3649.